roles/nginxplus/tasks/keys/apt-key.yml

```diff
@@ -6,15 +6,25 @@
     update_cache: true
 
 - name: "Install: NGINX Signing Key URL"
-  set_fact:
+  ansible.builtin.set_fact:
     default_keysite: >-
-      {{ (ansible_distribution_release == "trusty")
+      {{ (ansible_distribution_release == "jammy")
```

Comment:
Do we want to set the release to `jammy` everywhere? Isn't production running Focal?